ibm/service/kubernetes/resource_ibm_container_vpc_worker_pool.go

```diff
@@ -204,6 +205,13 @@ func ResourceIBMContainerVpcWorkerPool() *schema.Resource {
 				Description:      "Account ID of kms instance holder - if not provided, defaults to the account in use",
 				RequiredWith:     []string{"kms_instance_id", "crk"},
 			},
+
+			"import_on_create": {
+				Type:             schema.TypeBool,
+				Optional:         true,
+				DiffSuppressFunc: flex.ApplyOnce,
+				Description:      "Import a workerpool from a cluster",
```

Comment:
maybe a default:false for it?
the description can be improved a bit, maybe something like this?
```suggestion
				Description:      "Import an existing WorkerPool from the cluster, instead of creating a new",
```